docs/source/manual/jooq.rst

```diff
@@ -112,9 +112,85 @@ Below is an example of configuring the required binding uses a C3P0 pooling data
     }
   }
 
+Post commit actions
+~~~~~~~~~~~~~~~~~~~~
+
+Sometimes you need to perform an action only after the current transaction successfully commits. For example,
+updating or invalidating a cache.
+
+The JooqTransactionalHooks_ class provides the :code:`whenCommitted` method for registering callback actions that should be executed after a transaction is successfully committed. Any no-arguments lambda expression or :code:`Runnable` implementation can be passed as the action.
```

Comment:
Nit: There is an underscore after `JooqTransactionalHooks`

Reply:
It's a link down to below: https://github.com/cerner/beadledom/pull/140/files#diff-2a3927f61725d9baa48fb9c016e79882R196